setup.py

```diff
@@ -17,12 +17,27 @@
       description='the simple alchemistry test set',
       author='David Dotson',
       author_email='dotsdl@gmail.com',
+      maintainer='Oliver Beckstein',
+      maintainer_email='orbeckst@gmail.com',
       classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 4 - Beta',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: BSD License',
+        'Operating System :: POSIX',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows ',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
I think py35 is probably ok but it might be good to deprecate the py2?

Reply:
I want a release that is guaranteed to work with 2 and 3 (which it does). After then we can effectively adopt NEP29 (even though it currently doesn't matter for alchemtest).

alchemtest has no CHANGES or any other place for deprecations so I'll just write it in the short GitHub release notes that this will be the last release officially compatible with 2.7 and 3.5, 3.6.